examples/web/src/misc/LensClientInteroperability.tsx

```diff
@@ -0,0 +1,51 @@
+import { IStorageProvider, LensClient, development } from '@lens-protocol/client';
+import { useSession } from '@lens-protocol/react-web';
+import { useEffect, useState } from 'react';
+
+import { RequireProfileSession } from '../components/auth';
+
+class LocalStorageProvider implements IStorageProvider {
+  getItem(key: string) {
+    return window.localStorage.getItem(key);
+  }
+
+  setItem(key: string, value: string) {
+    window.localStorage.setItem(key, value);
+  }
+
+  removeItem(key: string) {
+    window.localStorage.removeItem(key);
+  }
+}
```

Comment:
Having the `LensProvider` using its own storage implementation and having this one accessing `localStorage` seems "uneven". Requires consumers to know something that until now was an internal.

Can we do something more encapsulated? Can we just use the same exact adapter instance that `@lens-protocol/react-web` use somehow?